### PR TITLE
chore: rename to use new ssbc URLs

### PR DESCRIPTION
**Pull request by decentral1se**
@staltz

### CODE REVIEW

File: go.mod
Comment (decentral1se) [inline go.mod:11]: Refs latest commit of https://github.com/ssbc/go-ssb-refs/pull/2 :+1: 

Probably we should do a proper release at some point but this works For Now :tm:

**Review by staltz — APPROVED** [review]
Much appreciated

(I'm waiting for CI tests before merging)

**Pull request merged**
